Simplenote/Classes/SPAuthViewController.swift

```diff
@@ -332,16 +381,21 @@ private extension SPAuthViewController {
 
 // MARK: - Actions
 //
-private extension SPAuthViewController {
+extension SPAuthViewController {
```

Comment:
Removing `private` since the AuthenticationMode descriptors live in its own file